### PR TITLE
Edits to how-to-contribute-to-appflowy draft

### PR DESCRIPTION
**Pull request by egp415**
* Added spacing for clarity
* Removed unecessary HTML formatting
* Ensured consistency of AppFlowy branding (i.e. case)
* Made sure heading hierarchy was consistent
* Fixed use of unordered lists
* Enforced stylistic rule for links (i.e. make links contain relevant text)
* Edited for clarity

### CODE REVIEW

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (JRS296) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:363]:
```suggestion
To summarize, we were able to understand how to contribute to AppFlowy, how to handle that first PR, how to connect with the community, and many others in this article.

Author: [Jonathan Rufus Samuel](https://github.com/JRS296)
Editors: [Eric Phillips](https://github.com/egp415) and [Alex Wallen](https://github.com/a-wallen)
```

**Review by JRS296 — CHANGES_REQUESTED** [review]
Few changes:
- Resolve spelling (line 18)
- Adding credits - Do I need to add a GitHub link? Or is there some other link that needs to be added? (in conclusion).

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (egp415) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:363]: I will leave it to @annieappflowy how she wants to handle attribution — I see no particular issue with GitHub links at this time, though I believe the idea is that it would be up to the individual author's preference.

I actually feel like the conclusion section needs a bit more work. The phrase "and many others" reads oddly and it lacks a good "call to action" such as linking once again to the list of items to work on.  I would start by rewriting it as follows:

In this article we discussed a variety of subjects such as how to contribute to AppFlowy, how to handle your first PR, and how to connect with the community.

And then something like:

We invite you to start contributing to AppFlowy by selecting your own [issue to work on](https://github.com/AppFlowy-IO/AppFlowy/issues).

Comment (egp415) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:363]: Yes, I think that sort of nudge would be helpful.

Comment (JRS296) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:363]: > 

Would this work? Let me know if any other changes are needed. Thanks.

Comment (egp415) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:363]: I would say now that you are now "over writing" — the conclusion should be much more concise.  It does not need to reiterate so much of the article's content.  Here is my suggestion on how to condense it down:
---
You should now be well-equipped to embark on your own open source journey. By following the guidelines in this article, you will be able to successfully resolve pull requests and engage meaningfully with other project contributors.

We hope you participate in AppFlowy's development and make an impact either through coding, bug reporting, feature suggestions, documentation. Contributing to this project and building relationships within the AppFlowy community not only fosters collaboration but also offers opportunities for learning and growth.

We invite you to start contributing to AppFlowy right away by selecting an [[issue to work on](https://github.com/AppFlowy-IO/AppFlowy/issues)](https://github.com/AppFlowy-IO/AppFlowy/issues).

Happy Coding!

Comment (JRS296) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:363]: This works much better. I think we ought to go ahead with your suggestion 👍

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (Xazin) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:18]: `solve` and `resolve` both sound a bit off, what about `How to identify and contribute with your first PR` or similar?

A bug can be resolved, but a feature request can be implemented.

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (Xazin) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:37]:
```suggestion
This is an open-source project, which means that its code is available to everyone for development and customization! Whether you're an aspiring developer, a seasoned programmer, or someone genuinely interested in technology, this guide is here to help you navigate the exciting world of open-source development with AppFlowy!
```

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (Xazin) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:50]:
```suggestion
3. How do I identify and open my first Pull Request?
```

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (Xazin) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:70]:
```suggestion
  - Read carefully through the <b>welcome</b> channel. You must abide by the server's rules and regulations.
```

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (Xazin) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:87]:
```suggestion
   - Use the [General Help Wanted](https://github.com/AppFlowy-IO/AppFlowy/discussions/categories/general-help-wanted) and [Technical Help Wanted](https://github.com/AppFlowy-IO/AppFlowy/discussions/categories/technical-help-wanted) section to find open projects for first time contributors.
```

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (Xazin) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:88]:
```suggestion
   - Explore AppFlowy's [6 Month Technical Roadmap](https://github.com/AppFlowy-IO/AppFlowy/discussions/1715) to gain a better understanding of AppFlowy's current development goals over the course of the next 6 months.
```

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (Xazin) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:98]: No reason to start using html links all of a sudden.

```suggestion
We recommend getting familiar with the [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow) for collaborating on projects and setting up your development environment.
```

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (Xazin) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:100]: Same
```suggestion
Once you're ready, start looking for a PR to work on. If you need help getting started, look at [AppFlowy's Documentation](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions).
```

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (Xazin) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:128]: Maybe add a note saying something about if there is no one assigned to the issue, don't worry about being assigned before starting work on it. Even if you aren't assigned, all pull requests that solve a GitHub issue is appreciated. 

Of course it's best to not work one something someone else is working on, but that's not something we can always control.

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (Xazin) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:135]:
```suggestion
As mentioned previously, we expect you to make contributions using the [GitHub Workflow](https://docs.github.com/en/get-started/quickstart/github-flow).
```

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (Xazin) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:140]: We can add a link to AppFlowy's documentation where the guide is present on how to setup AppFlowy locally.

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (Xazin) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:137]: We could use a step like "Familiarize with AppFlowy's code style, conventions, and guidelines" with one or more links.

Style Guide: https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides
Code Submission Guidelines: https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/code-submission-guidelines

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (Xazin) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:141]: Maybe another step, "Add unit tests for any new features where necessary"

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (Xazin) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:146]: 8. Wait for approval from the AppFlowy team, once approved then merge your branch
9. Delete your branch once the PR has been merged

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (egp415) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:150]: I added these links to a new section called "Before You Get Started" which also contains the link to creating a account and the GitHub Workflow

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (egp415) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:100]: in this case the subject is plural because they are two different sections referenced

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (egp415) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:141]: Maybe we could add caveat that you can work on issues that haven't been assigned to you but there is a greater possibility this will lead to merge conflicts should it end up being assigned to someone else.

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (egp415) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md]: Yes, I knew I shouldn't have trusted the auto-highlighting on my editor

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (egp415) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md]: changed it "open" as per your other suggestions

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (annieappflowy) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:141]: @JRS296 , would you like to add a note according to @Xazin and @egp415 suggestions?

Comment (JRS296) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:141]: Hello @annieappflowy, got caught up with something else, can I send in the note by tomorrow morning? Thanks.

Comment (JRS296) [inline essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:141]:
```suggestion
Send a message down in the comments section to get the issue assigned to yourself. Once that's done, you are ready to begin coding!

Note: 
- Don't worry about being assigned before starting work on it. Even if you aren't assigned, all pull requests that solve a GitHub issue is appreciated.
- However, please do understand that even though you can work on issues that haven't been assigned to you, there is an increased possibility that this will lead to merge conflicts should it end up being assigned to someone else. 
- You are also free to suggest improvements on an issue assigned to someone else. 
```

Comment (JRS296) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:141]: I hope this will do as far as the note is concerned. I feel it's better that we ask newcomers to message a member from the core team before getting started. I believe that is a part of the workflow for solving issues according to this article, so we ought to remain consistent.
As always, feel free to share your opinion on the same. @annieappflowy @egp415 @Xazin

**Review by JRS296 — APPROVED** [review]

File: essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md
Comment (egp415) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:141]: It should be "all pull requests that solve a GitHub issue are appreciated." (not "is appreciated")

I think it's worth discussing whether it's a good idea to suggest inexperienced developers / newcomers to the project start contributing before getting the go ahead per the process you have outlined — I can see it becoming more informal once someone has become an established contributor, but this is not necessarily something you  would want to even discuss within the context of this document.

Comment (JRS296) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:141]: Made the change 👍 

Yes, we could discuss it further. @annieappflowy and @Xazin your thoughts?

Comment (Xazin) [reply essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md:141]: I'm not sure if there is a demerit. Of course if someone is missing context or is inexperienced, that means the PR could be wrong. But asking "Please assign me" does not change that.

If a PR needs changes, or is wrong, then that is that. It can happen.

Opening a PR is not really destructive, I don't see the problem.

**Pull request merged**
